apps/explorer/src/explorer/ExplorerApp.tsx

```diff
@@ -143,8 +143,6 @@ export const ExplorerApp: React.FC = () => {
         <Router basename={process.env.BASE_URL}>
           <StateUpdaters />
           <Routes>
-            <Route path="/mainnet" element={<RedirectMainnet />} />
-            <Route path="/xdai" element={<RedirectXdai />} />
```

Comment:
Can you keep these?
It's there for historical reasons in case an old url is accessed.

Reply:
Do you think it is still relevant?
I've joined the team 1.5 years ago and have never seen urls with this prefixes.
Looks like it's super outdated

Reply:
Well, we were still creating those 😓 

The last one I could find is from ~1y ago:

https://cowservices.slack.com/archives/C03S445V822/p1673720749446989

Reply:
Ok ok, added it back